common/test/acceptance/tests/studio/test_import_export.py

```diff
@@ -0,0 +1,144 @@
+"""
+Acceptance tests for the Import and Export pages
```

Comment:
Thanks for doing this bok choy tests! It's so great having acceptance coverage of import/export.